dist/AppImageBuilder.yml

```diff
@@ -2,9 +2,11 @@ version: 1
 
 script:
 - |
-  icon_path="{{APPIMAGE_PATH}}/usr/share/icons/hicolor/128x128/apps"
+  cp -rl "{{APPIMAGE_PATH}}" "${TARGET_APPDIR}"
```

Comment:
How about just using `mv`? Any use of hard links makes me nervous, plus I'm not sure how this would interact with soft links.

Reply:
Actually, forget that, I'm fine with `cp -a` here to ensure that permissions/soft links are preserved. Performance impact from copying shouldn't be significant here anyways.

Reply:
No, I prefer `mv` but didn't use it because I had the impression that the target is existing before the `script` block. This is not the case, so `mv` is safer and more efficient IMHO.

Reply:
If it existed then it would copy into that directory even with `cp`. I'm not sure it's a good idea to be destructive here, but I'm not going to insist on either resolution.